phys2bids/slice4phys.py

```diff
@@ -107,6 +108,71 @@ def find_runs(phys_in, ntp_list, tr_list, thr=None, padding=9):
     return run_timestamps
 
 
+def slice_phys(phys, run_timestamps, time_before=5):
```

Comment:
@sangfrois Would you mind taking a look at this? My goal is to separate the slicing step from calling `find_runs`, since my workflow doesn't use that function. The modularization will also aid testing.